device_doctor/lib/src/health.dart

```diff
@@ -31,7 +31,7 @@ Future<HealthCheckResult> closeIosDialog({
   // Runs the single XCUITest in infra-dialog.
   await inDirectory(dialogDir, () async {
     List<String> command =
-        'xcrun xcodebuild -project infra-dialog.xcodeproj -scheme infra-dialog -destination id=$deviceId test'
+        'xcrun xcodebuild -project infra-dialog.xcodeproj -scheme infra-dialog -destination -quiet id=$deviceId test'
```

Comment:
stdout and stderr are thrown out a few lines later, tell Xcode not to bother to output anything with  `-quiet`.